src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/mappers/AddressesMapperTest.java

```diff
@@ -0,0 +1,154 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.bulk.scan.transformation.mappers;
+
+import com.google.common.collect.ImmutableMap;
+import org.junit.Test;
+import uk.gov.hmcts.reform.bsp.common.model.shared.in.OcrDataField;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.OcrFieldName;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.transformation.mappers.AddressesMapper;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiFunction;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class AddressesMapperTest {
+
+    public static final String LINE_1 = "AddressLine1";
+    public static final String LINE_2 = "AddressLine2";
+    public static final String TOWN = "AddressTown";
+    public static final String POSTCODE = "AddressPostcode";
+    public static final String COUNTY = "AddressCounty";
+    public static final String COUNTRY = "AddressCountry";
```

Comment:
Might be worth adding these to the TestConstants file so they can be reused elsewhere

Reply:
I was thinking about moving them to uk.gov.hmcts.reform.bsp.common.mapper.AddressMapper in bsp-common-lib as it's used over there as well:

```
private static ImmutableMap<String, String> getAddressMapping(String prefix) {
        String nestedFieldPrefix = StringUtils.capitalize(prefix + "Address");
        Map<String, String> address = new HashMap<>();

        address.put(nestedFieldPrefix + "Line1", "AddressLine1");
        address.put(nestedFieldPrefix + "Line2", "AddressLine2");
        address.put(nestedFieldPrefix + "County", "County");
        address.put(nestedFieldPrefix + "Postcode", "PostCode");
        address.put(nestedFieldPrefix + "Town", "PostTown");
        address.put(nestedFieldPrefix + "Country", "Country");

        return ImmutableMap.copyOf(address);
    }
```

then it could be re-used everywhere.

Reply:
https://github.com/hmcts/bsp-common-lib/pull/21